src/GameData.ts

```diff
@@ -157,7 +158,7 @@ export default class GameData {
             return;
         }
 
-        message.channel.send("```" + JSON.stringify(result, null, 4) + "```");
+        message.channel.send("```json\n" + JSON.stringify(result, null, 4) + "```");
```

Comment:
message.channel.send("```" + JSON.stringify(result, null, 4) + "```", {code:"json"});

Reply:
Then you'd have to leave out the backticks too. `message.channel.send(JSON.stringify(result, null, 4), {code:"json"})`. Both work, but IMO the first one is more clear what is actually sent.